namingserver/src/main/java/org/apache/seata/namingserver/manager/ClusterWatcherManager.java

```diff
@@ -88,18 +94,23 @@ public void onChangeEvent(ClusterChangeEvent event) {
     }
 
     private void notify(Watcher<?> watcher) {
-        notify(watcher, HttpServletResponse.SC_OK);
+        notify(watcher, HttpResponseStatus.OK.code());
     }
 
     private void notify(Watcher<?> watcher, int statusCode) {
-        AsyncContext asyncContext = (AsyncContext) watcher.getAsyncContext();
-        HttpServletResponse httpServletResponse = (HttpServletResponse) asyncContext.getResponse();
-        watcher.setDone(true);
+        Channel channel = (Channel) watcher.getAsyncContext();
+        if (channel instanceof Http2StreamChannel) {
+            ByteBuf buf = Unpooled.buffer(4);
```

Comment:
Why not `channel.writeAndFlush(new DefaultHttp2DataFrame(Unpooled.EMPTY_BUFFER));`